src/Table.js

```diff
@@ -44,16 +46,17 @@ const Table = (props) => {
     size ? 'table-' + size : false,
     bordered ? 'table-bordered' : false,
     striped ? 'table-striped' : false,
-    inverse ? 'table-inverse' : false,
+    (dark || inverse) ? 'table-dark' : false,
     hover ? 'table-hover' : false,
     reflow ? 'table-reflow' : false
   ), cssModule);
 
   const table = <Tag {...attributes} className={classes} />;
 
   if (responsive) {
+    const responsiveClassName = responsive === true ? 'table-responsive' : `table-${responsive}-responsive`;
```

Comment:
looks like it should be `table-responsive-${responsive}`.
Also looks like they killed the wrapper needed for responsive (something that was missing in alpha.6 upgrade)

Reply:
But the tests pass? :P wow, can't believe I missed that!